src/main/java/de/rototor/pdfbox/graphics2d/PdfBoxGraphics2D.java

```diff
@@ -477,11 +495,34 @@ public void drawGlyphVector(GlyphVector g, float x, float y) {
 	public void fill(Shape s) {
 		try {
 			contentStream.saveGraphicsState();
+
+			// Possibly set the alpha constant
+			if((composite != null) && (composite instanceof AlphaComposite))
```

Comment:
Would you mind removing the redundant composite != null check? 
null instanceof AlphaComposite will allways be false